go-controller/pkg/util/ovs.go

```diff
@@ -467,12 +429,6 @@ func RunOVNSbctl(args ...string) (string, string, error) {
 	return RunOVNSbctlWithTimeout(ovsCommandTimeout, args...)
 }
 
-// RunOVNCtl runs an ovn-ctl command.
-func RunOVNCtl(args ...string) (string, string, error) {
```

Comment:
this isn't windows specific. It might have some good use in the future like "status_ovnnb"

https://man7.org/linux/man-pages/man8/ovn-ctl.8.html

Maybe not worth removing?

Reply:
If we need it, we can bring it back. Has no current users.